.github/workflows/publish-tags.yml

```diff
@@ -1,157 +0,0 @@
-name: Publish to NPM beta
```

Comment:
This was pretty much a rename of publish-canary.

Again, git saw them as different files.